docs/related/amf-conversion.mdx

```diff
@@ -1,9 +1,193 @@
 ---
 id: amf_conversion
 title: AMF Conversion
-hide_title: true
 ---
 
-# AMF Conversion
+:::caution
+The behaviour expressed in this documentation is currently a work in progress.
+If any bugs are found that do not satisfy these requirements,
+please file an issue in the [aml-org/amf repository](https://github.com/aml-org/amf/pull/715).
+:::
 
+# What is WebAPIs conversion?
 
+Conversion stands for the capability of parsing an API from one specification vendor into an API graph and then rendering said graph as a different vendor’s specification.
+
+Different specifications can have different required fields, and the same information conceptually may not be described in the same way.
+This is the reason why in a *cross parsing-rendering process*, the AMF graph needs to go through a transformation called *“Compatibility resolution”*.
+
+**Compatibility resolution** is a specific resolution pipeline that is used to adapt an API graph parsed from spec A, to fulfill the requirements of the spec in which it will be rendered.
+An example of how to use Compatibility Resolution can be seen in [Example 1](#conversion-examples).
+
+
+How does this impact the API model? For example, fields that are not required in the origin spec but required in the target spec,
+will be added with default values if they are not present in the origin spec.
+
+:::tip The Objective
+The objective of conversion is to **output a valid AMF model for the spec in which it will be rendered**.
+It aims to **keep information loss to a minimum through a best-effort process** with some limitations.
+:::
+
+
+## Conversion Goals
+
+Conversion resolution has several goals:
+- Make transitioning to/from OAS to RAML easier for the end user.
+- Give the user an approximate view of what his API in another spec would look like
```

Comment:
We try to keep things gender-neutral if possible. 

Maybe:
- Give the user an approximate view of what **their** API in another spec would look like

Reply:
totally agree, nice catch.